test/unit/core/ShippingCountries.test.ts

```diff
@@ -15,9 +16,25 @@ this.questionableCountries = ['Europe', 'Asia'];
 
 describe('ShippingCountries', () => {
     test('isValidCountryCode() should return true for all country codes', () => {
+        // Check valid country codes
         for ( const x in this.countryCodeList ) {
             if ( x ) {
+                // Check country code is correct
                 expect(ShippingCountries.isValidCountryCode(x)).toBe(true);
+                // Check country from country code is correct
+                expect(ShippingCountries.isValidCountry(ShippingCountries.getCountry(x))).toBe(true);
+            }
+        }
+    });
+
+    test('isValidCountry() should return true for all country names', () => {
+        // Check valid contry names
+        for ( const x in this.countryList ) {
```

Comment:
prefer for of instead of for in
proper parameter name, x is not readable code
no need for if (x)